app/src/main/java/com/egaragul/androidfundametals/model/data/BaseGetRequest.kt

```diff
@@ -0,0 +1,9 @@
+package com.egaragul.androidfundametals.model.data
+
+import com.egaragul.androidfundametals.di.RetrofitModule
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class BaseGetRequest(
```

Comment:
why you need data class here
data class underhood generate toString,hashCode and equals methods 
I think here not need this method